### PR TITLE
tests: add full coverage for tox.session.cmd.run.common

### PR DESCRIPTION
**Pull request by gnikonorov**
**FOR THE REWRITE BRANCH**

Add full test coverage for `session/cmd/run/common.py`

## Contribution checklist:

(also see [CONTRIBUTING.rst](../tree/master/CONTRIBUTING.rst) for details)

- [x] wrote descriptive pull request text
- [x] added/updated test(s)
- [x] updated/extended the documentation - N/A
- [x] added relevant [issue keyword](https://help.github.com/articles/closing-issues-using-keywords/)
      in message body - N/A
- [x] added news fragment in [changelog folder](../tree/master/docs/changelog) - N/A
  * fragment name: `<issue number>.<type>.rst` for example (588.bugfix.rst)
  * `<type>` is must be one of `bugfix`, `feature`, `deprecation`,`breaking`, `doc`, `misc`
  * if PR has no issue: consider creating one first or change it to the PR number after creating the PR
  * "sign" fragment with "by :user:`<your username>`"
  * please use full sentences with correct case and punctuation, for example: "Fix issue with non-ascii contents in doctest text files - by :user:`superuser`."
  * also see [examples](../tree/master/docs/changelog)
- [x] added yourself to `CONTRIBUTORS` (preserving alphabetical order) -N/A

### CODE REVIEW

**Comment by gnikonorov** [comment]
This PR suffers from the same unrelated timeouts as in my previous testing PRs  ( `tests/session/cmd/run/test_int_setuptools.py::test_setuptools_package_py_project` ), @gaborbernat

File: src/tox/session/cmd/run/common.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ def __call__(
     ) -> None:
         value = "true" if values is None else values
         if value not in ("config", "true", "false"):
-            raise ArgumentTypeError("value must be config, true or false")
+            raise ArgumentTypeError("Value must be 'config', 'true', or 'false'. Got '{}'".format(value))
```
Comment (gaborbernat) [inline src/tox/session/cmd/run/common.py:17]: Don't capitalize exception messages.

File: tests/session/cmd/run/test_common.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+from argparse import ArgumentParser, ArgumentTypeError, Namespace
+from typing import Optional
+
+import pytest
+
+from tox.session.cmd.run.common import SkipMissingInterpreterAction
+
+
+@pytest.mark.parametrize("values", ["config", None])
+def test_call_SkipMissingInterpreterAction(values: Optional[str]) -> None:
```
Comment (gaborbernat) [inline tests/session/cmd/run/test_common.py:10]: Please keep test names snake case.

**Review by gaborbernat — CHANGES_REQUESTED** [review]

File: src/tox/session/cmd/run/common.py (hunk 1 above)
Comment (gaborbernat) [inline src/tox/session/cmd/run/common.py:17]: Also, use f-string instead of format.

File: tests/session/cmd/run/test_common.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    argument_parser = ArgumentParser(description="Test SkipMissingInterpreterAction.")
```
Comment (gaborbernat) [inline tests/session/cmd/run/test_common.py:11]: Don't add any information in tests that are not needed; e.g. here the description.

**Pull request merged**